### ISSUE TITLE
VSCode (Powershell) starts with two terminals open: Powershell Integrated and "pwsh" (cont'd)

### ISSUE THREAD

**Issue by joe-miller-hgs** [PowerShell/vscode-powershell#4471]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all [_open and closed_ issues](https://github.com/PowerShell/vscode-powershell/issues?q=is%3Aissue) to ensure it has not already been reported.
- [X] I have read the [troubleshooting](https://github.com/PowerShell/vscode-powershell/blob/main/docs/troubleshooting.md) guide.
- [X] I am sure this issue is with the _extension itself_ and does not reproduce in a standalone [PowerShell](https://github.com/PowerShell/PowerShell/issues/new/choose) instance.
- [X] I have verified that I am using the latest version of Visual Studio Code and the PowerShell extension.
- [ ] If this is a security issue, I have read the [security issue reporting guidance](https://github.com/PowerShell/vscode-powershell/blob/main/SECURITY.md).

### Summary

this is a continuation of [#3925](https://github.com/PowerShell/vscode-powershell/issues/3925).
it is not a new issue. it is an issue that has repeatedly been closed with no solution.

i am having the same problem as github-account1111.
when i open vscode powershell extension, i get two terminals:
![image](https://user-images.githubusercontent.com/84777053/226542321-351f2ace-64ac-4f90-87e7-89d2e8a32b2c.png)
and
![image](https://user-images.githubusercontent.com/84777053/226542367-284d21b3-c828-4f14-923a-34dcb68067a6.png)

i don't want two terminals.
this problem started when i installed pwsh.
the dual terminals only appear when i select pwsh in the session menu.
if i caused the problem somehow when i installed pwsh, i have no idea how and no idea how to fix it.

here's my session menu:
![image](https://user-images.githubusercontent.com/84777053/226542411-8294efc6-20a3-4db3-b188-dd137dabdbb4.png)

i prefer to have only the PSIC terminal when i open vscode with pwsh selected.
i don't need or want a duplicate terminal.
at minimum, it causes a big chunk of my terminal window to be consumed by the two-terminal list on the right, as shown in the first two images.
it also causes problems if i accidently close the wrong terminal when i try to restore my full terminal window.

what other info can i provide to help you diagnose this problem?
thanks for your time.

### PowerShell Version

```console
$PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.3.3
PSEdition                      Core
GitCommitId                    7.3.3
OS                             Microsoft Windows 10.0.19045
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Visual Studio Code Version

```console
.code --version
1.76.2
ee2b180d582a7f601fa6ecfdad8d9fd269ab1884
x64
```

### Extension Version

```console
.code --list-extensions --show-versions | Select-String powershell

ms-vscode.powershell@2023.2.1
```

### Steps to Reproduce

open vscode with pwsh selected

### Visuals

in original post

### Logs

[EditorServices.log](https://github.com/PowerShell/vscode-powershell/files/11026014/EditorServices.log)
[Language_Server_Protocol_payload_logs.log](https://github.com/PowerShell/vscode-powershell/files/11026018/Language_Server_Protocol_payload_logs.log)
[StartEditorServices.log](https://github.com/PowerShell/vscode-powershell/files/11026019/StartEditorServices.log)
[vscode-powershell.log](https://github.com/PowerShell/vscode-powershell/files/11026020/vscode-powershell.log)

**Comment by SydneyhSmith** [comment]
Thanks @joe-miller-hgs we do not have control over the creation of the pwsh terminal, you will need to ask VSCode to not create this terminal. If you would like to not see the extension terminal we have a setting for that:
"powershell.integratedConsole.showOnStartup": false

**Comment by andschwa** [comment]
@joe-miller-hgs I literally gave the same solution in the last post that you referenced: https://github.com/PowerShell/vscode-powershell/issues/3925#issuecomment-1102831347

**Comment by joe-miller-hgs** [comment]
@andschwa i disagree. the answer @SydneyhSmith gave was much clearer to someone who didn't already know the answer (like me)
